packages/machine/src/protocol/install-virtual-app.ts

```diff
@@ -1,11 +1,7 @@
 import { ETHVirtualAppAgreementCommitment } from "@counterfactual/machine/src/ethereum/eth-virtual-app-agreement-commitment";
 import { VirtualAppSetStateCommitment } from "@counterfactual/machine/src/ethereum/virtual-app-set-state-commitment";
-import {
-  AppInterface,
-  AssetType,
-  NetworkContext,
-  SolidityABIEncoderV2Struct
-} from "@counterfactual/types";
+import { virtualAppKey } from "@counterfactual/machine/src/virtual-app-key";
```

Comment:
Should export it from the module vs diving into the code

Reply:
it is exported - fixing the import path